docs/SETTINGS.rst

```diff
@@ -14,6 +14,7 @@ Bootstrap configuration
     -  **loop\_wait**: the number of seconds the loop will sleep. Default value: 10
     -  **ttl**: the TTL to acquire the leader lock. Think of it as the length of time before initiation of the automatic failover process. Default value: 30
     -  **maximum\_lag\_on\_failover**: the maximum bytes a follower may lag to be able to participate in leader election.
+    -  **synchronous\_mode**: turns on synchronous replication mode. Synchronous mode trades service availability for better durability guarantees. See `replication modes documentation <https://github.com/zalando/patroni/blob/master/docs/replication_modes.rst>`__ for details.
```

Comment:
A replica does not exactly run in synchronous mode, it's a cluster wide thing. I reworded it to be about lost transactions versus availability for writes and added a sentence about the mechanism. This should give a decent overview for both technically inclined and newbie users.